pages/platform/databases/postgresql_09_concept_high_availability/guide.de-de.md

```diff
@@ -106,12 +106,9 @@ No intervention is required from the customer.
 
 OVHcloud also detects when a whole datacenter is lost. As a user, your cluster will still be visible in API and Control Panel.
 
-As of writing, **automatic backups are stored in the same datacenter location**. In case of Datacenter failure, OVHcloud cannot perform a restoration process.
-A restoration process can be launched manually by the customer if manual backups (pg_dump) are available. It will require the creation of a new cluster, configuring users, authorized IPs, ...
-
-We plan to provide backups storage in another location during the 2022 calendar year.
-If the automatic backups are stored in another datacenter location, it will be possible from API or UI to restore a previously made automatic backup. You will be able to select the right one and restore it in the datacenter of your choice.
-It's a manual intervention, that will also create a new Service URI. You will need to modify all the applications that were using the previous (and now unavailable) Service URI.
+Automatic backups are first performed on-site (i.e. in the same region as the service), then replicated off-site, to another datacenter.
+In case of Datacenter failure, the recovery strategy involves creating a new service in another region, either using the fork from backup feature from an automatic backup that was replicated off-site, or restoring from backups performed manually (e.g. `pg_restore` a previously performed `pg_dump`).
```

Comment:
Hello @rgbj, I'm having issues with this sentence. Not sure the following suggestion is better but you may want to rephrase it then.
```suggestion
In case of a Datacenter failure, the recovery strategy involves creating a new service in another region, either using the fork (from the backup feature) from an automatic backup that was replicated off-site, or restoring from backups performed manually (e.g. `pg_restore` as previously performed `pg_dump`).
```